opg-php-fpm-71-ppa-1604/Dockerfile

```diff
@@ -16,58 +39,34 @@ ARG PHP_PACKAGES="mcrypt \
                   php7.1-pgsql \
                   php7.1-xml"
 
+
 RUN apt-get update -y && \
+# Add ondrej PPA and instal above packages
```

Comment:
`install` if you fancy correcting a typo

Reply:
thank :)